bike.js

```diff
@@ -0,0 +1,145 @@
+const Table = require('cli-table2')
+
+
+async function _getUserBikedToWorkDistance(state, userId) {
+  const rows = await state.db.query('SELECT * FROM users WHERE id = $1', [userId + ''])
```

Comment:
What is this part used for `[userId + ' ']` ?

Reply:
`userId` is a number here but a text field in the DB. Query didn't work. Doing `variable + ''` is a JS trick to convert the value to a string, and with that change the query works.